cmd/subnetcmd/publish.go

```diff
@@ -146,6 +146,36 @@ func doPublish(sc *models.Sidecar, subnetName string, publisherCreateFunc newPub
 		return err
 	}
 
+	if noRepoPath != "" {
+		ux.Logger.PrintToUser("Writing the file specs to the provided directory at: %s", noRepoPath)
+		// the directory does not exist
+		if _, err := os.Stat(noRepoPath); err != nil {
+			if err := os.MkdirAll(noRepoPath, constants.DefaultPerms755); err != nil {
+				return fmt.Errorf("attempted to create the given --no-repo-path directory at %s, but failed: %w", noRepoPath, err)
+			}
+			ux.Logger.PrintToUser("The given --no-repo-path at %s was not existing; created it with permissions %o", noRepoPath, constants.DefaultPerms755)
+		}
+		subnetFile := filepath.Join(noRepoPath, subnetName+constants.YAMLSuffix)
+		vmFile := filepath.Join(noRepoPath, vm.Alias+constants.YAMLSuffix)
+		if !forceWrite {
+			// do not automatically overwrite
+			if _, err := os.Stat(subnetFile); err == nil {
+				return fmt.Errorf("a file with the name %s already exists. If you wish to overwrite, provide the %s flag", subnetFile, forceFlag)
+			}
+			if _, err := os.Stat(vmFile); err == nil {
+				return fmt.Errorf("a file with the name %s already exists. If you wish to overwrite, provide the %s flag", vmFile, forceFlag)
+			}
+		}
+		if err := os.WriteFile(subnetFile, subnetYAML, constants.DefaultPerms755); err != nil {
```

Comment:
These need to be in a `vms` and `subnets` directory

![Screen Shot 2022-09-02 at 4 12 38 PM](https://user-images.githubusercontent.com/3431073/188229232-01b8e730-72f9-4729-81fe-27aa015579bb.png)

Reply:
ok fixed